timesketch/lib/datastores/elastic.py

```diff
@@ -912,14 +926,15 @@ def import_event(self, index_name, event_type, event=None, event_id=None,
 
         return self.import_counter['events']
 
-    def flush_queued_events(self):
+    def flush_queued_events(self, retry_count=0):
```

Comment:
Why have retry_count as an argument? Does it get set from a caller somewhere?

Reply:
it is called by the exception handling, only there, to be a `recursive` function